src/components/common/ColonyHome/ColonyMembersWidget/MembersSubsection.tsx

```diff
@@ -160,7 +160,6 @@ const MembersSubsection = ({
                 // banned={canAdministerComments && banned}
                 banned={false}
                 showInfo
-                notSet={false}
```

Comment:
UserAvatar had notSet default to true, even though more or less everywhere it was used it was being passed a false prop, like here. I have removed the default and removed these props where they were used.